smoke-tests/build.gradle

```diff
@@ -74,6 +74,9 @@ android {
 repositories {
   google()
   jcenter()
+  maven {
+    url "../build/m2repository/"
```

Comment:
Maybe add a comment describing what is in this repository and how it is created?

Reply:
Done.